flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/testutils/SemanticTestBase.java

```diff
@@ -135,6 +130,25 @@ void runTests(TableTestProgram program) throws Exception {
         }
     }
 
+    private static Map<String, String> createSourceOptions(String id) {
+        final Map<String, String> options = new HashMap<>();
+        options.put("connector", "values");
+        options.put("data-id", id);
+        options.put("runtime-source", "NewSource");
+        // Enforce per-record watermarks for testing
+        options.put("disable-lookup", "true");
+        options.put("enable-watermark-push-down", "true");
+        options.put("scan.watermark.emit.strategy", "on-event");
+        return options;
+    }
+
+    private static Map<String, String> createSinkOptions() {
+        final Map<String, String> options = new HashMap<>();
+        options.put("connector", "values");
+        options.put("sink-insert-only", "false");
+        return options;
+    }
```

Comment:
since now minimum is jdk11
how about replace with `Map.of` or `Map.ofEntries` ?